election-server/src/bin/http.rs

```diff
@@ -245,6 +281,34 @@ async fn main() -> anyhow::Result<()> {
         None
     };
 
+    (prometheus_layer, prometheus_handle)
+}
+
+/// Configures the HTTP server which serves as an API for election components.
+/// Returns a handle for the corresponding process spawned, or an error if
+/// configuration fails.
+async fn setup_http(
+    config: &AppConfig,
+    prometheus_layer: PrometheusLayer,
+) -> Result<tokio::task::JoinHandle<Result<(), anyhow::Error>>, anyhow::Error> {
+    let state = AppState {
+        db_pool: DatabasePool::create(config.db_connection.clone(), config.pool_size, true)
+            .await
+            .context("Failed to connect to the database")?,
+    };
+    let frontend_config: FrontendConfig = config.into();
+
+    // Render index.html with config
+    let index_template = std::fs::read_to_string(config.frontend_dir.join("index.html"))
+        .context("Frontend was not built.")?;
+    let mut reg = Handlebars::new();
+    // Prevent handlebars from escaping inserted object
+    reg.register_escape_fn(|s| s.into());
```

Comment:
This escape function does nothing. Is there some default escaping?

Reply:
I'll look into it (copied from web3id).

Reply:
there was a convenient `no_escape` function you could supply to this, which is what is recommended in the docs.